README.md

```diff
@@ -155,6 +155,9 @@ Modules from the community have not been tested by the Lilypad team. Run these m
 | [Llama 3.2 3B](https://github.com/rhochmayr/ollama-llama3.2-3b/tree/1.0.0)           | `lilypad run github.com/rhochmayr/ollama-llama3.2-3b:1.0.0`      | `-i PROMPT=""` |
 | [Qwen 2.5 Coder 7B](https://github.com/rhochmayr/ollama-qwen2.5-coder-7b/tree/1.0.0) | `lilypad run github.com/rhochmayr/ollama-qwen2.5-coder-7b:1.0.0` | `-i Prompt=""` |
 
+| [Spark-TTS-0.5B](https://github.com/jamiebones/Spark-TTS-0.5B/releases/tag/v1.3) | `lilypad run github.com/jamiebones/Spark-TTS-0.5B.git:v1.3  --web3-private-key <private-key> -i input="Text to convert to speech"` |
```

Comment:
This is formatted incorrectly to the rest.

Here is the correct formatting: 
```
| [Spark-TTS-0.5B](https://github.com/jamiebones/Spark-TTS-0.5B/releases/tag/v1.3) | `lilypad run github.com/jamiebones/Spark-TTS-0.5B.git:v1.3` | `-i input=""` |
```

Also I'm seeing in your repo you also have `prompt_text` and `prompt_speech_path`... What is `prompt_text` used for and are both of these usable in the module? If so, could you please add them to the inputs section along with `input`?